src/api/Flooq.Api/Controllers/LinearizedGraphController.cs

```diff
@@ -44,26 +56,38 @@ public async Task<ActionResult<LinearizedGraph>> GetGraph(Guid id)
         }
 
         // POST: api/LinearizedGraph
-        // To protect from overposting attacks, see https://go.microsoft.com/fwlink/?linkid=2123754
+        /// <summary>
+        /// Adds a <see cref="LinearizedGraph"/>.
+        /// </summary>
+        /// <param name="linearizedGraph">The new <see cref="LinearizedGraph"/>.</param>
+        /// <returns>
+        /// A <see cref="CreatedAtActionResult"/> object that produces a <see cref="StatusCodes.Status201Created"/> response
+        /// or <see cref="ConflictResult"/> if the <see cref="LinearizedGraph"/> already exists.
+        /// </returns>
         [HttpPost]
         public async Task<ActionResult<LinearizedGraph>> PostGraph(LinearizedGraph linearizedGraph)
         {
-            if (LinearizedGraphExists(linearizedGraph.Id))
+            _graphService.AddGraph(linearizedGraph);
+            try
             {
-              _graphMetricsService.IncrementBadRequestCount();
```

Comment:
This should be added in the catch

Reply:
I think this will be done by @silvan-flum in #156.